nautobot/extras/plugins/marketplace_manifest.yml

```diff
@@ -94,6 +94,24 @@
+- "name": "DNS Models"
```

Comment:
I didn't see the whole file but are these alpha sorted?

Reply:
Yes! The tooling handles that for us, and this is between `Circuit Maintenance Tracker` and `Data Validation Engine` (I guess capitals sort before lowercase letters).